tests/load_data/event_data/test_tracab.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+
+from databallpy.load_data.metadata import Metadata
+from databallpy.load_data.tracking_data.tracab import (
+    _get_meta_data,
```

Comment:
Not all functions have individual tests, this is necesssary, also if you already get 100% covered. If an error emerges it might be hard to say if a specific function fails or if the flow is wrong (for instance the input given to the next function is wrong). It will be a lot easier to find mistakes if all unique functions have their own tests. So also add a test for `_get_player_data`. I think you would make it yourself a lot easier if you input in that function a list of dictionaries for every player instead of the bs4.element.Tag.